server/routes/users.js

```diff
@@ -22,27 +22,24 @@ router.get('/profile', isAuthenticated, (req, res) => {
       columns: ['id', 'username', 'name', 'email', 'address']
     })
     .then(users => {
+      console.log(users)
       res.json(users);
     })
     .catch(err => {
       res.json(err);
     })
 });
 
-router.put('/users', isAuthenticated, (req, res) => {
+router.put('/edit-profile', isAuthenticated, (req, res) => {
```

Comment:
this route should be on `/profile`

Reply:
Wanted to double check with you if it should be /users since its on the readme.  I changed it to /users for now because I wasnt too sure.